IS7024/Pages/LudlowGarage.cshtml.cs

```diff
@@ -35,8 +35,8 @@ public void OnGet()
                 if (jsonObject.IsValid(schema, out validationEvents))
                 {
                     var rootObject = RootObject.FromJson(jsonString);
-                    List<Event> events = rootObject.ResultsPage.Results.Event.ToList();
-                    ViewData["events"] = events;
+                    List<Event> eventsList = rootObject.ResultsPage.Results.Event.ToList();
+                    ViewData["events"] = eventsList;
```

Comment:
Changed "events" to eventsList to indicate that the variable is a List

Reply:
I'm good with events here.  The plural nature of the variable indicates that it is a collection.